compact/ecal_forward_homogeneous.xml

```diff
@@ -0,0 +1,92 @@
+<lccdd>
+
+  <info name="ecal_forward_homogeneous.xml"
+        title="Homogeneous forward ECal"
+        author="@rymilton"
+        url="https://github.com/rymilton"
+        status="development"
+        version="1.0"
+  ><comment/></info>
+
+  <define>
+    <documentation>
+      #### Material Thickness
+    </documentation>
+    <constant name="EcalEndcapPCoverThickness"            value="0.1*cm"/>
+    <constant name="EcalEndcapPAirThickness"              value="10.15*cm"/>
+    <constant name="EcalEndcapPPCBThickness"              value="0.25*cm"/>
+    <constant name="EcalEndcapPLightGuideThickness"       value="2.5*cm"/>
+    <constant name="EcalEndcapPScintillatorThickness"     value="17.*cm"/>
+
+    <documentation>
+      - Ecal Endcap N Layers and computed Thickness
+    </documentation>
+    <constant name="EcalEndcapPSingleLayerThickness"
+      value = "EcalEndcapPCoverThickness +
+               EcalEndcapPAirThickness +
+               EcalEndcapPPCBThickness +
+               EcalEndcapPLightGuideThickness +
+               EcalEndcapPScintillatorThickness "
+    />
+    <constant name="EcalEndcapPLayer_NRepeat" value="EcalEndcapP_numLayers"/>
+  </define>
+
+  <limits>
+  </limits>
+
+  <regions>
+  </regions>
+
+  <display>
+  </display>
+
+  <detectors>
+
+    <documentation>
+      ### Forward (Positive Z) Endcap Electromagnetic Calorimeter
+
+      Simplified homogeneous implementation of the forward ScFi calorimeter
+      Each layer contains an aluminum cover, an air gap, PCB, light guides, and scintillating material
+
+      Insert shape is cut out from middle of detector
+    </documentation>
+    <detector id="ECalEndcapP_ID"
+      name="EcalEndcapP"
+      type="epic_EndcapCalorimeterWithInsert"
+      readout="EcalEndcapPHits">
+      <position x="0" y="0" z="EcalEndcapP_zmin"/>
+      <dimensions
+        rmin="EcalEndcapP_rmin"
+        rmax="EcalEndcapP_rmax"
+        z="EcalEndcapPSingleLayerThickness * EcalEndcapPLayer_NRepeat"
+      />
+      <layer repeat="EcalEndcapPLayer_NRepeat" thickness = "EcalEndcapPSingleLayerThickness">
+        <slice name="Cover_slice" material="Aluminum" thickness="EcalEndcapPCoverThickness" vis="AnlRed"/>
+        <slice name="Air_slice" material="Air" thickness="EcalEndcapPAirThickness" vis="AnlRed"/>
+        <slice name="PCB_slice" material="Fr4" thickness="EcalEndcapPPCBThickness" vis="AnlRed"/>
+        <slice name="LightGuide_slice" material="Plexiglass" thickness="EcalEndcapPLightGuideThickness" vis="AnlBlue"/>
+        <slice name="Scintillator_slice" material="AvgTungstenScFi" thickness="EcalEndcapPScintillatorThickness" vis="AnlOrange" sensitive="true"/>
+      </layer>
+      <insert>
+        <position x="EcalEndcapPInsert_xposition" y="0" z = "0"/>
```

Comment:
```suggestion
        <position x="EcalEndcapPInsert_xposition" y="0" z="0"/>
```

Reply:
Thanks for finding this format typo. I removed the space.